metrics/metrics.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package metrics contains expvar & Prometheus types and code used by
+// Tailscale for monitoring.
+package metrics
+
+import "expvar"
+
+// Map is a string-to-Var map variable that satisfies the expvar.Var
+// interface.
+//
+// Semantically, this is mapped by tsweb's Prometheus exporter as a
```

Comment:
Probably for a separate change, but: some of the stuff currently exported by derper and stun is ripe for using a map as I described above. May send you a change to implement that before I wire derper up into Prometheus (so we have a clean tsdb).

Reply:
SGTM